app/src/main/java/com/github/epfl/meili/profile/friends/FriendsListRecyclerAdapter.kt

```diff
@@ -1,30 +1,56 @@
 package com.github.epfl.meili.profile.friends
 
+import android.util.Log
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.Button
+import android.widget.ImageView
```

Comment:
Unused import

Reply:
Sorry about that I usually don't forget to do this. Thanks for pointing it out!